pkg/monitoring/services/services_windows.go

```diff
@@ -18,7 +18,7 @@ var ErrorNotImplementedForOS error
 type Win32_Service struct {
 	Name             string
 	DisplayName      string
-	Description      string
+	Description      *string
```

Comment:
Why do we have to make it a pointer?

Reply:
because of this
![image](https://user-images.githubusercontent.com/1025231/52069166-e0ddfa00-2586-11e9-9925-83a18fed5331.png)